dts/arm/nordic/nrf9160_common.dtsi

```diff
@@ -47,7 +55,7 @@ egu0: egu@1b000 {
 egu1: egu@1c000 {
 	compatible = "nordic,nrf-egu";
 	reg = <0x1c000 0x1000>;
-	interrupts = <28 NRF_DEFAULT_IRQ_PRIORITY>;
```

Comment:
What what this value?

Reply:
This was introduced in 1.5.0 instead of having hard-coded values. It is set to 1 in `zephyr/dts/arm/nordic/nrf_common.dtsi`

Reply:
`NRF_DEFAULT_IRQ_PRIORITY` is set to 1 in `dts/arm/nordic/nrf_comon.dtsi`